packages/web/babel.ts

```diff
@@ -18,7 +18,6 @@ module.exports = function (api: any, options: {}) {
           // Do not transform modules to CJS
           modules: false,
           targets: api.env('legacy') ? undefined : { esmodules: true },
-          bugfixes: true,
```

Comment:
No optimization for `module/nomodule` pattern?
https://babeljs.io/blog/2020/03/16/7.9.0#babel-preset-env-s-bugfixes-option-11083-https-githubcom-babel-babel-pull-11083

Reply:
It's not supported in babel 7.8.3. We're having some weird CI failures in our recent PRs after upgrading to babel 7.9 so I thought I'd reset it back to 7.8.3. Doesn't seem to be help though so will try upgrading again and see why the CI the failing.

Reply:
Ah, I see. Thank you for your explanation.